conf/nonclangable.conf

```diff
@@ -1,3 +1,18 @@
+#linux-ld: ls-caps-vendor.o: relocation R_RISCV_HI20 against `a local symbol' can not be used when making a shared object; recompile with -fPIC
+#| clang-14: error: unable to execute command: Segmentation fault (core dumped)
+#| clang-14: error: linker command failed due to signal (use -v to see invocation)
+TOOLCHAIN:pn-pciutils:riscv64 = "gcc"
+
+#| riscv64-yadro-linux-ld: erl_bits.c:(.text+0xc60): undefined reference to `__extendhfsf2'
+#| riscv64-yadro-linux-ld: erl_bits.c:(.text+0x1a8a): undefined reference to `__truncdfhf2'
+TOOLCHAIN:pn-erlang:riscv64 = "gcc"
+
+#| linux-yocto/5.15.72+gitAUTOINC+c6aba7f07a_0b628306d1-r0/temp/run.do_uboot_mkimage.47933: 185: riscv64-yadro-linux-llvm-objcopy: not found
+OBJCOPY:pn-linux-yocto:riscv64:toolchain-clang = "${HOST_PREFIX}objcopy"
+
+#babeltrace2: ELF binary /usr/lib/babeltrace2/plugins/babeltrace-plugin-text.so has relocations in .text [textrel]
+TOOLCHAIN:pn-babeltrace2 = "gcc"
+TOOLCHAIN:pn-u-boot = "gcc"
```

Comment:
this change I have in a different review too https://github.com/kraj/meta-clang/pull/697